test-result-summary-client/src/TabularView/TabularView.jsx

```diff
@@ -485,7 +487,7 @@ export default class TabularView extends Component {
         let found = false;
 
         data.forEach(function (testResultsObject) {
-            platform = testResultsObject.buildName
+            platform = testResultsObject.buildName.split("_").slice(1).join('_').replace("sanity.perf_", "");
```

Comment:
That doesn't look right! Suppose the pipeline name is `Test_openjdk8_j9_sanity.perf_x86-64_linux_Liberty`, then `buildName.split("_").slice(1).join('_').replace("sanity.perf_", "")` will give us `openjdk8_j9_x86-64_linux_Liberty`. We just want to keep index 4 and 5 so that we get `x86-64_linux`, which becomes the column name for Tabular View. Other info is not needed since it's shown elsewhere already.

Reply:
Agree with that we should not hard coded 'sanity.perf', but do we need to keep the jdkversion and jvm type: openjdk8/11_j9 for showing in the column name?

Reply:
Nope! We don't need to keep the JDK version or type since those are already mentioned in the dropdown menu. Hence, all columns would have the same JDK version and type. We should only show architecture and os (i.e. index 4 and 5).

Reply:
We already have code to get the platform and other variables. Please do not reinvent the wheel.
https://github.com/AdoptOpenJDK/openjdk-test-tools/blob/6eade0055a35a953dbd38693541228b5c94061bd/test-result-summary-client/src/Build/Summary/ResultSummary.jsx#L56